themes/bootstrap3/js/cart.js

```diff
@@ -201,7 +214,7 @@ VuFind.register('cart', function Cart() {
       }
       _cartNotificationTimeout = setTimeout(function notificationHide() {
         $('#' + elId).popover('hide');
-      }, 5000);
+      }, 50000);
```

Comment:
Oops, that wasn't an intentional change, though 5 secs is low when there's a lot going on. Reverted now since that's something to think about another time.